crates/node/src/commands/run.rs

```diff
@@ -2,7 +2,13 @@ use std::path::PathBuf;
 
 use clap::ValueHint::FilePath;
 use madara_runtime::SealingMode;
-use mc_data_availability::DaLayer;
+use mc_data_availability::avail::config::AvailConfig;
+use mc_data_availability::avail::AvailClient;
+use mc_data_availability::celestia::config::CelestiaConfig;
+use mc_data_availability::celestia::CelestiaClient;
+use mc_data_availability::ethereum::config::EthereumConfig;
+use mc_data_availability::ethereum::EthereumClient;
+use mc_data_availability::{DaClient, DaLayer};
```

Comment:
When you rebase you will have to add `features = ["avai", "celestia"]` to your import of `mc_data_availability`